.github/actions/system-info/action.yml

```
@@ -0,0 +1,24 @@
+name: system-info
+description: Get system information
+
+outputs:
+  platform:
+    description: The platform name
```

It would be nice to have examples of possible values in the descriptions, like
```suggestion
    description: The platform name ("linux", "windows", "macos")
```

---

done